src/components/admin/DataTable.vue

```diff
@@ -1,60 +1,49 @@
 <template>
   <q-table
+    hide-bottom
```

Comment:
adding this removes the pagination preventing from viewing the content on the next page